test/test_pp_api.py

```diff
@@ -452,21 +452,44 @@ def test_chat_tools_input(self):
             assert False, f"Error while trying to pass in tools to chat template: {e}"
 
     def test_qwen2_5_vl_chat_template(self):
-        model_id = "Qwen/Qwen2.5-VL-72B-Instruct"
+        model_id = "Qwen/Qwen2.5-1.5B-Instruct"
+        
+        # For Qwen, tools should be a list of functions stored in json format, passed in
+        # separately, as an additional input to apply_chat_template
+        # (for Phi-4, it should be part of messages)
+        tools = [
+            {
+                "name": "get_match_result",
+                "description": "get match result",
+                "parameters": {
+                    "match": {
+                        "description": "The name of the match",
+                        "type": "str",
+                        "default": "Arsenal vs ManCity"
+                    }
+                }
+            },
+        ]
+
+
         messages = [
+            {
+                "role": "system",
+                "content": "You are a helpful assistant",
+                # "tools": json.dumps(tools), # pass the tools into system message using tools argument
```

Comment:
Should the commented debug be removed?

Reply:
I think we should keep it, especially in this Extensions test file; Kunal had the same comment - I think it's useful for understanding the qwen/phi-4 tools usage actually - I added another comment near the top of the test to give more context.